primitive-types/impls/serde/benches/impl_serde.rs

```diff
@@ -55,3 +59,74 @@ fn u256_to_hex(c: &mut Criterion) {
 		),
 	);
 }
+
+fn hex_to_u256(c: &mut Criterion) {
+	let parameters = vec![
+		r#""0x0""#,
+		r#""0x1""#,
+		r#""0x10""#,
+		r#""0x100""#,
+		r#""0x1000000000000000000000000000000000000000000000000000000000000100""#,
+	];
+
+	c.bench(
+		"hex_to_u256",
+		ParameterizedBenchmark::new(
+			"",
+			|b, x| {
+				b.iter(|| {
+					black_box(serde_json::from_str::<U256>(&x))
+				})
+			},
+			parameters
+		)
+	);
+}
+
+fn bytes_to_hex(c: &mut Criterion) {
+	let parameters = vec![
+		serde_json::from_str::<Bytes>(&input::HEX_64_CHARS).unwrap(),
+		serde_json::from_str::<Bytes>(&input::HEX_256_CHARS).unwrap(),
+		serde_json::from_str::<Bytes>(&input::HEX_1024_CHARS).unwrap(),
+		serde_json::from_str::<Bytes>(&input::HEX_4096_CHARS).unwrap(),
+		serde_json::from_str::<Bytes>(&input::HEX_16384_CHARS).unwrap(),
+		serde_json::from_str::<Bytes>(&input::HEX_65536_CHARS).unwrap(),
+	];
+
+	c.bench(
+		"bytes to hex",
+		ParameterizedBenchmark::new(
+			"",
+			|b, x| {
+				b.iter(|| {
+					black_box(serde_json::to_string(&x))
+				})
+			},
+			parameters
+		)
+	);
+}
+
+fn hex_to_bytes(c: &mut Criterion) {
+	let parameters = vec![
+		input::HEX_64_CHARS,
+		input::HEX_256_CHARS,
+		input::HEX_1024_CHARS,
+		input::HEX_4096_CHARS,
```

Comment:
I don't think we need to measure values above 1024 here. Having thee datapoints are enough to ensure it scales linearly.

Reply:
ok, I tried locally it was slower, I can remove bigger input if you think it's sufficient